disco/voice/client.py

```diff
@@ -150,6 +234,10 @@ def __init__(self, channel, encoder=None):
         self.timestamp = 0
 
         self.update_listener = None
+        self.ssrc_lookup = {}
```

Comment:
You need to hook up VoiceStateUpdate and clear the ssrc when a user disconnects, not only will this grow indefinitely but it also has the following race;

- User joins
- User speaks
- User leaves
- User rejoins

SSRC is now invalid until the user speaks again.